src/main/java/seedu/contax/storage/JsonAdaptedTag.java

```diff
@@ -45,4 +45,16 @@ public Tag toModelType() throws IllegalValueException {
         return new Tag(tagName);
     }
 
+    @Override
+    public boolean equals(Object o) {
+        if (o == this) {
+            return true;
+        }
+
+        if (o instanceof JsonAdaptedTag) {
+            return ((JsonAdaptedTag) o).getTagName().equals(tagName);
+        }
+
+        return false;
```

Comment:
Small nitpick, but I would follow the guard-clause coding guideline.
```suggestion
        if (!(o instanceof JsonAdaptedTag)) {
            return false;
        }

        return ((JsonAdaptedTag) o).getTagName().equals(tagName);
```

Reply:
Fixed in 2ee5c01f